src/_includes/partials/stylesheets.njk

```diff
@@ -1,6 +1,11 @@
+{% set cssPattern = r/.*\.css$/g %}
+{% for key, value in assets %}
+{% if cssPattern.test(key) %}
+	<link href="/assets{{ value }}" rel="stylesheet" />
+{% endif %}
+{% endfor %}
```

Comment:
@greatislander I'm running into a strange issue here. I was testing out the processing of multiple style files. I used both the same file type and different file types. What appears to be happening is that some of the files are being missed when expanding the partial. The files all appear in the dist directory but references aren't all showing up in the `<link>` tags in the html page. I'm not even sure how it is missing files as it doesn't depend on the exact name, nor the alphabetical order. It might also just use 2 of them, as that's the most I've gotten to appear so far.

Reply:
I believe this is resolved now, @jobara (see 6e23937).